src/test/resources/features/complaints/ComplaintsEndToEnd.feature

```diff
@@ -21,6 +22,16 @@ Feature: Complaints End To End
       | Ex-Gratia        |
       | Minor Misconduct |
 
+  Scenario Outline: User moves a BF case to the Triage stage
+    Given I am logged into "CS" as user "BF_USER"
+    When I create a "BF" case and move it to the "<complaintType> Triage" stage
```

Comment:
Is this the right step? 
I think if you wanted to test for both complaint types by the stage name doesnt vary between them you should instead use the iCreateACOMPCaseForAComplaintAndMoveItToStage step